release/nightly_tests/stress_tests/test_dead_actors.py

```diff
@@ -85,7 +91,7 @@ def parse_script_args():
                 ray.cluster_resources())
 
     parents = [
-        Parent.remote(num_children, death_probability)
+        Parent.remote(num_children, death_probability, num_threads)
```

Comment:
I think it should be

`Parent.options(max_concurrency=num_threads).remote(...)`